src/routes/Loading.tsx

```diff
@@ -169,48 +169,40 @@ export function Loading(props: IProps) {
 
     //filter will be passed to api which will lowercase the email before comparison
     var allinvites: Invitation[] = (await newremote.query((q: QueryBuilder) =>
-      q
-        .findRecords('invitation')
-        .filter(
-          { attribute: 'email', value: userEmail },
-          { attribute: 'accepted', value: false }
-        )
+      q.findRecords('invitation').filter(
+        { attribute: 'email', value: userEmail }
+        // { attribute: 'accepted', value: false }  //went from AND to OR between attributes :/
```

Comment:
does it accept q.findRecords('invitation').filter({attribute:'email', value: userEmail}).filter({attribute: 'accepted', value:false})
(using chaining for AND)